metrics/ci_integrations/docs/diagrams/config_validator_sequence_diagram.puml

```diff
@@ -54,14 +54,14 @@ alt auth is valid
 
     CoolIntegrationClient --> CoolIntegrationDestinationValidationDelegate : InteractionResult.success
     CoolIntegrationDestinationValidationDelegate -> CoolIntegrationDestinationValidationDelegate : performs additional validation
-    CoolIntegrationDestinationValidationDelegate --> CoolIntegrationDestinationValidator : InteractionResult.success
-    CoolIntegrationDestinationValidator -> ValidationResultBuilder : setResult(CoolIntegrationConfigField.auth, FieldValidationResult.success())
+    CoolIntegrationDestinationValidationDelegate --> CoolIntegrationDestinationValidator : authFieldValidationResult
```

Comment:
```suggestion
    CoolIntegrationDestinationValidationDelegate --> CoolIntegrationDestinationValidator : FieldValidationResult.success
```
Would it be a bit more clear? If so, please, apply this suggestion to all similar situations.